packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -234,20 +236,48 @@ export const buildAggregatedFilterItemsApplier = (
     return null;
   }
 
-  return (row, shouldApplyFilter) => {
-    const resultPerItemId: GridFilterItemResult = {};
-
-    for (let i = 0; i < appliers.length; i += 1) {
-      const applier = appliers[i];
-      if (!shouldApplyFilter || shouldApplyFilter(applier.item.field)) {
-        resultPerItemId[applier.item.id!] = applier.v7
-          ? applier.fn(row)
-          : applier.fn(getRowId ? getRowId(row) : row.id);
-      }
-    }
-
-    return resultPerItemId;
-  };
+  // Original logic:
+  // return (row, shouldApplyFilter) => {
+  //   const resultPerItemId: GridFilterItemResult = {};
+  //
+  //   for (let i = 0; i < appliers.length; i += 1) {
+  //     const applier = appliers[i];
+  //     if (!shouldApplyFilter || shouldApplyFilter(applier.item.field)) {
+  //       resultPerItemId[applier.item.id!] = applier.v7
+  //         ? applier.fn(row)
+  //         : applier.fn(getRowId ? getRowId(row) : row.id);
+  //     }
+  //   }
+  //
+  //   return resultPerItemId;
+  // };
```

Comment:
I generally don't preserve dead code in comments but in this case it feels like it helps readability of the next section. I could remove if there is opposition.

Reply:
Yeah, the string template is indeed hard to read and the original code was helpful to me 👍